coopertunes/hparams/Audio2Mel.py

```diff
@@ -0,0 +1,20 @@
+from pathlib import Path
+from typing import Any, Optional, Union
+
+from coopertunes.hparams import HParams
+
+
+class Audio2MelHParams(HParams):
```

Comment:
Generally if you need it only for MelGan it could be in MelGan hparams

Reply:
It could be used for any other training when online spectrogram calculation  could be benefitial (adding noise masks, random crops etc.)